packages/component-library/src/components/form/mt-colorpicker/mt-colorpicker.interactive.stories.ts

```diff
@@ -2,6 +2,7 @@ import { within, userEvent, fireEvent } from "@storybook/test";
 import { expect } from "@storybook/test";
 
 import meta, { type MtColorpickerMeta, type MtColorpickerStory } from "./mt-colorpicker.stories";
+import { waitUntil } from "@/_internal/test-helper";
```

Comment:
There's a `waitFor` function exported by `@storybook/test` could you use that over this function?

Reply:
Don't know it yet but sounds good 👍

Reply:
I just looked at the `waitFor` method. Their implementation excepts a error everytime when it should get retried. This would make the code much longer. Here is a comparison:
```js
    await waitUntil(() => args.updateModelValue.mock.calls.length > 0);
    // vs
    await waitFor(() => {
      if (args.updateModelValue.mock.calls.length > 0) {
        return;
      }

      throw new Error("Model value was not updated");
    });
  ```
  
  So I think the `waitUntil` method is much cleaner and shorter